specs/subresourceintegrity/index.html

```diff
@@ -761,6 +761,11 @@ <h4 id="handling-integrity-violations">Handling integrity violations</h4>
 or execute resources that fail an integrity check, <em>but</em> MUST
 <a href="http://www.w3.org/TR/CSP2/#report-a-violation">report a violation</a>.</p>
 
+    <p class="note">On a failed integrity check, an <code>error</code> event is thrown. Developers
+wishing to provide a canonical fallback resource can catch this
```

Comment:
hmm ..sorry to be sweating details so much but canonical fallback resource might not mean much to people. Since this is a note, maybe even expand that bit? "canonical fallback resource (e.g., a resource not served from a CDN)" or words to that effect?